### PR TITLE
Posts filter on survey question view - #DEVAS-2194

### PR DESCRIPTION
**Pull request by tdesvenain**

### CODE REVIEW

File: assembl/static2/js/app/pages/survey.jsx [hunk 1]
```diff
@@ -99,14 +99,15 @@ class Survey extends React.Component<Props, State> {
...
-    questions.forEach((question) => {
-      if (question.posts.edges.length > 0) isProposals = true;
-    });
-    return isProposals;
-  };
+  //
+  // getIfProposals = (questions) => {
+  //   if (!questions) return false;
+  //   let isProposals = false;
+  //   questions.forEach((question) => {
```
[the remark this answers is not in the document]
Comment (tdesvenain) [reply assembl/static2/js/app/pages/survey.jsx]: ok

File: assembl/static2/js/app/components/debate/thread/threadView.jsx [hunk 2]
```diff
@@ -50,9 +57,10 @@ class ThreadView extends React.Component<Props> {
         ) : null}
         <Grid fluid className="background-grey">
           <div id="thread-view" className="max-container background-grey">
-            <PostsFilterMenu />
+            <ThreadPostsFilterMenu stickyOffset={60} stickyTopPosition={200} />
```
Comment (gary-bluenove) [inline assembl/static2/js/app/components/debate/thread/threadView.jsx]: The 60 and 200 are fixed values? This ok on responsive screens

File: assembl/static2/js/app/pages/survey.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gary-bluenove) [reply assembl/static2/js/app/pages/survey.jsx]: Je suis d'ac avec @felixsaintgenis

File: assembl/static2/js/app/utils/translations.js [hunk 3]
```diff
@@ -1538,7 +1543,12 @@ const Translations = {
         postSuccess: "Thanks for your participation. Your proposal has been sent!",
         postToBeValidated:
           "Your contribution has been saved. To insure the quality of the debate, it will be visible to all participants once it has been reviewed by the animation team.",
-        endPhase: "The \"%{closedPhaseName}\" phase is closed."
+        endPhase: "The \"%{closedPhaseName}\" phase is closed.",
+        postsOrder: {
+          chronologicalTop: "Chronological points of view",
```
Comment (gary-bluenove) [inline assembl/static2/js/app/utils/translations.js]: Not sure I'd use 'points' here but I suspect it's already the standard term in assembl. We would normally just call it a post.

File: assembl/static2/js/app/components/debate/thread/threadView.jsx (hunk 2 above)
Comment (tdesvenain) [reply assembl/static2/js/app/components/debate/thread/threadView.jsx]: They are, i first set them in css, but this was not very maintainable. This allows to change it easily when we add a new filter menu or when we move it. This allows us to compute it when necessary.

Comment (tdesvenain) [reply assembl/static2/js/app/components/debate/thread/threadView.jsx]: I have tested it on responsive screen, it is ok (it is about height, not width)

File: assembl/static2/js/app/utils/translations.js (hunk 3 above)
Comment (tdesvenain) [reply assembl/static2/js/app/utils/translations.js]: You may be right, but the labelling was specified in the ticket.

**Comment by tdesvenain** [comment]
@gary-bluenove @ayyazdaniaryan on dirait que ces commits ne sont toujours pas sur uat. ai-je fait une boulette, oublié quelque chose ? pourriez vous jeter un oeil aux assets pour voir ?

**Comment by gary-bluenove** [comment]
Je vais deployer 3.2.1 sur UAT maintenant

Le mer. 4 sept. 2019 à 15:32, Thomas Desvenain <notifications@github.com> a
écrit :

**Pull request merged**